src/components/rootcomponents/Input.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+interface InputProps {
+  id: string;
+  type: string;
+  placeholder: string;
+  className?: string;
+}
+
+export const Input: React.FC<InputProps> = ({
+  id,
+  type,
+  placeholder,
+  className,
+  ...rest
+}) => {
+  return (
+    <div className={`mb-4 ${className}`}>
```

Comment:
I think we might need this className more on the input than here. Or you can add another one